functional_tests/catboost/test_catboost.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+"""Test new catboost integration."""
```

Comment:
It will become not-so-new eventually... :)
```suggestion
"""Test CatBoost integration."""
```